### PR TITLE
added exasol community adapter documentation

### PR DESCRIPTION
**Pull request by tglunde**
We added an adapter for the Exasol Analytics database. These are the corresponding documentation sites.

### CODE REVIEW

**Comment by clrcrl** [comment]
@jtcohen6 — is this `current` or `next`?

**Comment by jtcohen6** [comment]
This is `current`, since it looks like the exasol adapter [is pinned to dbt v0.14.0](https://github.com/tglunde/dbt-exasol/blob/master/setup.py#L27). I'll assign myself as a reviewer.

**Comment by tglunde** [comment]
> This is `current`, since it looks like the exasol adapter [is pinned to dbt v0.14.0](https://github.com/tglunde/dbt-exasol/blob/master/setup.py#L27). I'll assign myself as a reviewer.

Could you please explain the relevance on current versus next? Which versions include current/next?

**Comment by jtcohen6** [comment]
> Could you please explain the relevance on current versus next? Which versions include current/next?

Surely! The `next` branch serves a prerelease docs site, which will include any features/changes coming in a dbt version that has yet to reach final release.

Since the dbt-exasol plugin is usable _without_ needing to wait for v0.18.0, this PR can be against `current`, not `next`.

**Comment by tglunde** [comment]
when will this get reviews?

File: website/docs/reference/warehouse-profiles/exasol-profile.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
```
Comment (drewbanin) [inline website/docs/reference/warehouse-profiles/exasol-profile.md:1]: i think you're going to need to add this link [to the `sidebars.js` file](https://github.com/fishtown-analytics/docs.getdbt.com/blob/current/website/sidebars.js#L422) to get the sidebar to render [on this page](https://deploy-preview-291--docs-getdbt-com.netlify.app/reference/warehouse-profiles/exasol-profile).

**Review by drewbanin — CHANGES_REQUESTED** [review]
Hey @tglunde - thanks for the bump. This one was blocking on my review!

I have one (small) suggested change here. Happy to merge it once that change is addressed. Thanks for your hard work on this plugin and for your patience with this PR!

File: website/docs/reference/warehouse-profiles/exasol-profile.md (hunk 1 above)
Comment (tglunde) [reply website/docs/reference/warehouse-profiles/exasol-profile.md:1]: sorry, but I don't know how - I just copied another file and changed the content - i dont see any sidebar.js link on the postgres-profile.md either. could you give me a hint on how to do this?

Comment (drewbanin) [reply website/docs/reference/warehouse-profiles/exasol-profile.md:1]: all good! It's an easy thing to miss.

You're just going to want to update this file to add a link to `exasol-profile`:

https://github.com/fishtown-analytics/docs.getdbt.com/blob/3deda0efd5e6f4c1b3ca06ea0178818edd9f563c/website/sidebars.js#L415-L422

Check out the "sidebar" I'm referring to here:

Screen Shot 2020-08-21 at 9 25 31 AM

Comment (tglunde) [reply website/docs/reference/warehouse-profiles/exasol-profile.md:1]: ok. thx. added the commit.

**Review by drewbanin — APPROVED** [review]
Thanks @tglunde! Approved :)

**Pull request merged**